peazip-sources/peach.pas

```diff
@@ -57038,6 +57048,11 @@ procedure setbasevars;
 if executable_path<>'' then
    if executable_path[length(executable_path)]<>directoryseparator then executable_path:=executable_path+directoryseparator;
 setcurrentdir(executable_path);
+{$IFDEF DARWIN}
+   resource_path:=executable_path+'../Resources/';
+{$ELSE}
+   resource_path:=resource_path;

```

Comment:
Structure is 
(peazip app folder)/ contains peazip binary
(peazip app folder)/res/contains resources, so - in non Darwin build - resource_path needs to be
resource_path:=executable_path+'res'+directoryseparator